File: C#/p4.cs
//taking user input String , number

using System;

class p4{
static void Main(String[] args){
//String input
Console.WriteLine("Enter your name : ");
String name = Console.ReadLine();

//int input
Console.WriteLine("Enter Your Age : ");
int age = Convert.ToInt32(Console.ReadLine());

Console.WriteLine("Student Deatils are : \n name : " + name + " \n age : " + age );
}
}

File: C++/c++ lab programs/p4.cpp
#include<iostream>
using namespace std;

int main(){
	int n,m,i,s=0;
	cout<<"Enter the number: ";
	cin>>n;
	m=n;
    while(n>0){
    	i=n/10;
    	s=s*10+i;
    	n=n/10;
	}	
	if(m==s){
		cout<<endl<<"The number is palantrom";
		
	}
	else {
		cout<<endl<<"The number is not a palantrom";
	}
	return 0;
}
